src/site/twitch.tv/modules/hd-video/HDVideoModule.vue

```diff
@@ -0,0 +1,55 @@
+<template />
+
+<script setup lang="ts">
+import { log } from "@/common/Logger";
+import { declareModule } from "@/composable/useModule";
+import { useConfig } from "@/composable/useSettings";
+import { declareConfig } from "@/composable/useSettings";
+import { watchEffect } from "vue";
+
+const { markAsReady } = declareModule("hd-video", {
+	name: "HD Video",
+	depends_on: [""],
+});
+
+const enableHDVideo = useConfig<boolean>("general.hd-video.enabled");
+
+watchEffect(() => {
+	if (enableHDVideo.value) {
+		try {
+			document.addEventListener(
+				"visibilitychange",
+				(e) => {
+					e.stopImmediatePropagation();
```

Comment:
This is not a good solution as it would prevent any other `visibilitychange` listeners from functioning. This includes one used by the Extension to pause chat in an out-of-focus state.